src/main/java/com/bootcamp/demo/business/AccountService.java

```diff
@@ -30,45 +29,60 @@ public void setEncryptor(Encryptor encryptor) {
         this.encryptor = encryptor;
     }
 
-    public boolean accountValidation(Account account){
+    public String accountValidation(Account account) throws ServiceException {
+
+        StringBuilder stringBuilder = new StringBuilder();
         if(account.getName().isEmpty() || account.getPassword().isEmpty() || account.getEmail().isEmpty() || account.getUsername().isEmpty() || account.getAddress().isEmpty() || account.getPhoneNumber().isEmpty() || account.getDateOfBirth().isEmpty())
-            return false;
+            stringBuilder.append("Empty fields");
 
         String expression = "^([A-Z][a-z]*((\\s)))+[A-Z][a-z]*$";
         if(!account.getName().matches(expression))
-            return false;
+            stringBuilder.append("Wrong name format: must contain only letters and must be \"FirstName LastName\"  \n");
 
         expression = "[a-zA-Z0-9]*$";
```

Comment:
do not use same object for different patterns. Use dedicated ones and declare them as constants